lib/ws_servers/api/handlers/on_algo_order_remove.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+const sendError = require('../../../util/ws/send_error')
+const validateParams = require('../../../util/ws/validate_params')
+const isAuthorized = require('../../../util/ws/is_authorized')
+
+module.exports = async (server, ws, msg) => {
+  const { d, algoDB } = server
+
+  const [, authToken, exID, orders] = msg
+
+  const validRequest = validateParams(ws, {
+    exID: { type: 'string', v: exID },
+    authToken: { type: 'string', v: authToken },
+    orders: { type: 'array', v: orders }
+  })
+
+  if (!validRequest) {
+    d('invalid request: algo:remove')
+    return
+  }
+
+  if (!isAuthorized(ws, authToken)) {
+    return sendError(ws, 'Unauthorized')
+  } else if (!ws.aoc) {
+    return sendError(ws, 'Unauthorized')
+  }
+
+  for (const algOrder of orders) {
+    const { gid, algoID } = algOrder
+    const { AlgoOrder } = algoDB
+    await AlgoOrder.update({ gid, algoID }, { active: false })
```

Comment:
does it work on runtime? 

should we stop the algo order first on the algo host?